samples/excel/85-preview-apis/culture-info-date-time.yaml

```diff
@@ -0,0 +1,138 @@
+order: 5
+id: excel-culture-info-date-time
+name: 'Culture info: date and time'
+description: This sample shows how to apply the cultural settings APIs to retrieve read-only operating system date and time settings.
+host: EXCEL
+api_set:
+    ExcelAPI: '1.12'
+script:
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#display-date-time-setting").click(() => tryCatch(displayDateTimeSetting));
+        $("#write-date-time-setting").click(() => tryCatch(writeDateTimeSetting));
+
+        /* Create table with only header content. */
+        async function setup() {
```

Comment:
Nit: The `setup` method is typically after the featured methods of the snippet (right above `tryCatch`).

Reply:
That way seems weird to me because the methods don't follow the same order in which they're triggered at the top of the file, but I think consistency across samples is important, so I will change this.